app/assets/src/components/ui/controls/SearchBox.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import { Search } from "semantic-ui-react";
+import { escapeRegExp, debounce } from "lodash";
+
+const delayCheckMatch = 1000;
```

Comment:
Lets just add this inside the class as variables.

Reply:
Or capitalize as constants.

Reply:
Ok, moved it inside the class